lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -343,8 +343,10 @@ def stack_operation(cfn, stack_name, operation):
             else:
                 ret.update({'changed': False, 'failed': True, 'output' : 'Stack not found.'})
                 return ret
-        elif stack['StackStatus'].endswith('_ROLLBACK_COMPLETE'):
-            ret.update({'changed': True, 'failed' :True, 'output': 'Problem with %s. Rollback complete' % operation})
+        # it covers ROLLBACK_COMPLETE, UPDATE_ROLLBACK_COMPLETE and UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS
```

Comment:
This won't cover `UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS` since it's checking that the string ends with "ROLLBACK_COMPLETE" which is intentional. The cleanup in progress state isn't an end condition.